src/view.c

```diff
@@ -34,6 +34,12 @@ enum view_edge {
 	VIEW_EDGE_CENTER,
 };
 
+static bool
+is_enabled_output(struct output *output)
+{
+	return output && output->wlr_output->enabled;
```

Comment:
I think this should also check for `!output->leased`. See output.c for reference.
Should we move this function to `output.c` and call it `output_available()` or something instead?

Reply:
Sounds good. Would `output_is_usable` be a good name or do you prefer `output_available`?

Reply:
`output_is_usable` sounds usable to me